x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/alert_context_menu.tsx

```diff
@@ -52,14 +50,6 @@ interface AlertContextMenuProps {
   timelineId: string;
 }
 
-const addExceptionModalInitialState: AddExceptionModalBaseProps = {
-  ruleName: '',
-  ruleId: '',
-  ruleIndices: [],
-  exceptionListType: 'detection',
-  alertData: undefined,
-};
-
 const AlertContextMenuComponent: React.FC<AlertContextMenuProps> = ({
   disabled,
   ecsRowData,
```

Comment:
Is it possible to remove `nonEcsRowData` as a prop in this component now? I don't know if we're still using it.

Reply:
I was able to replace the use of `nonEcsRowData` where we were using it with `ecsRowData`. In testing it appears all expected fields existed in `ecsRowData`.

Reply:
EDIT: misunderstood what you meant. Good catch, removed `nonEcsRowData` as a prop.